Makefile

```diff
@@ -0,0 +1,13 @@
+all: dev
+
+serve:
+	hugo serve
+
+dev:
+	hugo serve --watch=true --disableFastRender -D
+
+install:
+	git submodule update --init --recursive
+	@which hugo && echo "Please install Hugo extended: https://gohugo.io/installation/"
+
+.PHONY: serve dev install
```

Comment:
In this repo we previously used the scripts from `package.json` for this purpose, but either approach is fine for me.

@HarshCasper do you have a preference to either one of them?

Reply:
sure, feel free to share your preference or remove it @HarshCasper 

I find it helpful to have a generic `Makefile` entry point as in most of our repositories to get started quickly.